client/my-sites/media/main.jsx

```diff
@@ -204,6 +205,25 @@ class Media extends Component {
 		this.maybeRedirectToAll();
 	};
 
+	updateItem = ( itemId, detail ) => {
+		const { selectedItems } = this.state;
+		const index = findIndex( selectedItems, ( item ) => item.ID === itemId );
```

Comment:
```suggestion
		const index = selectedItems.findIndex( ( item ) => item.ID === itemId );
```
Do consider native `Array.findIndex` since we're trying to move away from lodash? (See p4TIVU-9Bf-p2 and  https://github.com/Automattic/wp-calypso/pull/51976)